terracommon/tropp/serializers.py

```diff
@@ -106,30 +106,34 @@ class Meta(CampaignSerializer.Meta):
 
 class PictureSerializer(serializers.ModelSerializer):
     owner = serializers.ReadOnlyField(source='owner.email')
+    file = VersatileImageFieldSerializer('tropp')
 
     class Meta:
         model = Picture
         fields = '__all__'
 
 
 class SimplePictureSerializer(PictureSerializer):
-    file = VersatileImageFieldSerializer('tropp')
```

Comment:
We may keep this for frontend ?

Reply:
Yes, it's already in `PictureSerializer` :)

Reply:
Yes but the ViewpointSerializerWithPicture use the simple one

Reply:
Yes, and the simple one is a subclass of PictureSerializer :
` class SimplePictureSerializer(PictureSerializer)`
So the definition of `file` should be correct.

Reply:
Ok, that the information I didn't see !